github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/help.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Trigger that run job for GitHub Pull Request activities.
+    Trigger that launches builds in response to GitHub Pull Request activities.
```

Comment:
I mean it integrates only with 2 features. GH Issues (comments, labels)  and GH PRs (commit hash changed, PR opened, PR closed).